ddtrace/bootstrap/preload.py

```diff
@@ -6,6 +6,7 @@
 import os  # noqa:I001
 
 from ddtrace import config  # noqa:F401
+from ddtrace.appsec._iast._utils import _is_iast_enabled
```

Comment:
I saw in a different file there's a DEV comment saying:
> DEV: These imports are here to avoid _ast.ast_patching import in the top level because they are slow and affect serverless startup time

I know this is a different import, but is importing this and using `_is_iast_enabled` faster than just looking up the env var directly here? If it is please disregard this comment, just wasn't sure 🫣

Reply:
Great question! The goal of this PR is to allow at some point to use `configure` to enable IAST, regardless of env vars.
In any case, I just timed them to be sure:
```
$ ddtrace-run -d python -m timeit "import os; os.getenv('DD_IAST_ENABLED', False)"
500000 loops, best of 5: 574 nsec per loop
```
vs
```
$ ddtrace-run -d python -m timeit "from ddtrace.appsec._iast._utils import _is_iast_enabled; _is_iast_enabled()"
500000 loops, best of 5: 466 nsec per loop
```